### PR TITLE
1.6.0 release update conversion table

### PR DESCRIPTION
**Pull request by poodlewars**
#### Checklist

<details>
  <summary>
   Checklist for code changes...
  </summary>
 
 - [ ] Have you updated the relevant docstrings and documentation?
 - [ ] Is this contribution tested against [all ArcticDB's features](../docs/mkdocs/docs/technical/contributing.md)?
 - [ ] Do all exceptions introduced raise appropriate [error messages](https://docs.arcticdb.io/error_messages/)?
 - [ ] Are API changes highlighted in the PR description?
 - [ ] Is the PR labelled as enhancement or bug so it appears in autogenerated release notes?
</details>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,6 +142,8 @@ Please see version conversion dates in the below table:
 | 1.2 | Business Source License 1.1 | May 22, 2025 |
 | 1.3 | Business Source License 1.1 | Jun  9, 2025 |
 | 1.4 | Business Source License 1.1 | Jun  23, 2025 |
+| 1.5 | Business Source License 1.1 | Jul  11, 2025 |
+| 1.6 | Business Source License 1.1 | Jul  25, 2025 |
```
Comment (jjerphan) [inline README.md:146]: Was it released effectively yesterday?

File: docs/mkdocs/docs/licensing.md [hunk 2]
```diff
@@ -12,12 +12,7 @@ Once a version of ArcticDB has converted to the Apache 2.0 license, it may be us
 
 ## License conversion timeline
 
-| ArcticDB Version | License | Converts to Apache 2.0 |
-| ------------- | ------------- | ------------- |
-| 1.0 | Business Source License 1.1 | Mar 16, 2025 |
-| 1.2 | Business Source License 1.1 | May 22, 2025 |
-| 1.3 | Business Source License 1.1 | Jun  9, 2025 |
-| 1.4 | Business Source License 1.1 | Jun  23, 2025 |
+This is stored in the [README.md](https://github.com/man-group/ArcticDB/blob/master/README.md) of the project.
```
Comment (jjerphan) [inline docs/mkdocs/docs/licensing.md:15]: Thank you for removing duplication!

File: docs/mkdocs/docs/technical/releasing.md [hunk 3]
```diff
@@ -55,9 +55,7 @@ on `main`.
 
 ## 5. Update the BSL conversion table
 
-If you are releasing either a major (e.g. 2.0.0) or a minor (e.g. 1.5.0) release, please update the license conversion schedule tables:
-
-- [in the readme](https://github.com/man-group/ArcticDB/blob/master/README.md)
-- [and in the licensing documentation](https://github.com/man-group/ArcticDB/blob/master/docs/mkdocs/docs/licensing.md)
+If you are releasing either a major (e.g. 2.0.0) or a minor (e.g. 1.5.0) release, please update the license conversion
```
Comment (jjerphan) [inline docs/mkdocs/docs/technical/releasing.md:58]: Nitpick.
```suggestion
If you are releasing either a major (e.g. 2.0.0) or a minor (e.g. 1.5.0) version, please update the license conversion
```

**Review by jjerphan — APPROVED** [review]
Thank you, @poodlewars.

**Review by mehertz — APPROVED** [review]

**Pull request merged**